### DISCUSSION TITLE
Can’t scroll fields in CP on iPad

### DISCUSSION THREAD

**Discussion by kaitwalla** [statamic/cms#7247] [Q&A]
I'm trying to edit my site on my iPad, and while the left sidebar scrolls just fine, when I go to add a field and get the listing of the various field types, it's locked to view and I can't view anything below the cutoff point.

**Comment by duncanmcclean** [comment]
Would you be able to take a screenshot of where specifically you're running into the issue?

**Reply by kaitwalla** [reply]
![image](https://user-images.githubusercontent.com/3107030/208263558-e7c2af46-065c-4a99-bc26-ef84cc4df82b.png)
